cogs/modmail.py

```diff
@@ -80,30 +80,43 @@ async def _close_generic(self, user, guild, channel, delay):
 
     @app_commands.command(name='reply', description='Replys to a modmail, with your username')
     @app_commands.describe(content='The message to send to the user')
+    @app_commands.describe(attachment='An image or file to send to the user')
     @app_commands.guild_only()
     @app_commands.default_permissions(view_audit_log=True)
-    async def _reply_user(self, interaction: discord.Interaction, content: app_commands.Range[str, None, 1800]):
+    async def _reply_user(
+        self,
+        interaction: discord.Interaction,
+        content: app_commands.Range[str, None, 1800],
+        attachment: typing.Optional[discord.Attachment],
+    ):
 
-        await self._reply(interaction, content)
+        await self._reply(interaction, content, attachment)
 
     @app_commands.command(name='areply', description='Replys to a modmail, anonymously')
     @app_commands.describe(content='The message to send to the user')
+    @app_commands.describe(attachment='An image or file to send to the user')
     @app_commands.guild_only()
     @app_commands.default_permissions(view_audit_log=True)
-    async def _reply_anon(self, interaction: discord.Interaction, content: app_commands.Range[str, None, 1800]):
+    async def _reply_anon(
+        self,
+        interaction: discord.Interaction,
+        content: app_commands.Range[str, None, 1800],
+        attachment: typing.Optional[discord.Attachment],
+    ):
 
-        await self._reply(interaction, content, True)
+        await self._reply(interaction, content, attachment, True)
 
-    async def _reply(self, interaction: discord.Interaction, content, anonymous=False):
+    async def _reply(self, interaction: discord.Interaction, content, attachment, anonymous=False):
         db = mclient.modmail.logs
         doc = db.find_one({'channel_id': str(interaction.channel.id)})
         # Attachments are unable to be sent mod -> user with slash commands (unless it's a url).
         #
         # attachments = [x.url for x in ctx.message.attachments]
         # TODO: If ever able, reenable this functionality
-        attachments = []
-        # if not content and not attachments:
-        #    return await ctx.send('You must provide reply content, attachments, or both to use this command')
+        if not content and not attachment:
+            return await interaction.response.send_message(
+                'You must provide reply content, attachments, or both to use this command', ephemeral=True
```

Comment:
Could flow better if written as "To use this command, you must provide either context, an attachment, or both"